README.md

```diff
@@ -1,4 +1,4 @@
-# MPI-CorrBench &middot; [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause) 
+# MPI-CorrBench 1.1.0 &middot; [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause) 
```

Comment:
I personally would remove the dot, but do not have a strong opinion.

Reply:
Do we want to add a CI badge for the visualization (maybe with the embedded link to the dashboard)?

Reply:
That's a cool idea, but I would wait for it until the dashboard is online.